novawallet/Modules/Vote/Governance/ReferendumDetails/ReferendumDetailsInteractor.swift

```diff
@@ -291,12 +295,44 @@ final class ReferendumDetailsInteractor: AnyCancellableCleaning {
             metadataProvider?.refresh()
         }
     }
+
+    private func provideAbstains() {
+        guard let votersLocalWrapperFactory else {
+            return
+        }
+
+        let wrapper = votersLocalWrapperFactory.createWrapper(
+            for: .init(referendumId: referendum.index, votersType: .abstains)
+        )
+
+        abstainsFetchCancellable = wrapper
+
+        execute(
+            wrapper: wrapper,
+            inOperationQueue: operationQueue,
+            runningCallbackIn: .main
+        ) { [weak self] result in
+            guard let self, abstainsFetchCancellable === wrapper else {
+                return
+            }
+
+            clear(cancellable: &abstainsFetchCancellable)
+
+            switch result {
+            case let .success(voters):
+                presenter?.didReceiveAbstains(voters)
+            case let .failure(error):
+                presenter?.didReceiveError(.accountVotesFailed(error))
+            }
+        }
+    }
 }
 
 extension ReferendumDetailsInteractor: ReferendumDetailsInteractorInputProtocol {
     func setup() {
         makeSubscriptions()
         updateActionDetails()
+        provideAbstains()
```

Comment:
I would trigger abstain fetching each time referendum updates (see subscribeReferendum). It allows us to make sure they will be eventually up to date